src/resources/js/localization.ts

```diff
@@ -4,16 +4,22 @@ export const strings = new LocalizedStrings({
   en: {
     loginVerb: 'Log In',
     loginTitle: 'Login',
+    emailTitle: 'Email',
     usernameTitle: 'Username',
     passwordTitle: 'Password',
     rootServerTitle: 'Root Server',
     dashboardTitle: 'Dashboard',
     meetingsTitle: 'Meetings',
     serviceBodiesTitle: 'Service Bodies',
     usersTitle: 'Users',
+    userTitle: 'User',
```

Comment:
Try to find translations for each of these strings from the existing translation files in the root server - I sent you the github url to those files in Slack.